src/main/java/com.marketdata.api/MarketDataController.java

```diff
@@ -10,13 +8,22 @@
 @Controller
 public class MarketDataController {
 
+    /**
+     * index page
+     * @return
+     */
     @RequestMapping("/")
     public String index() {
         return "index";
     }
 
+    /**
+     *
+     * @param symbol
+     * @return returns ticker symbol
+     */
     @GetMapping("/MarketData/{symbol}")
     public String fetchSymbol(@PathVariable("symbol") String symbol) {
-        return "index";
+        return symbol;
```

Comment:
Felt like index was a placeholder and this wasn't returning anything.

Reply:
Keep this as is.  It should return the name of a page to render, which would be index.html.  symbol is a parameter, and will have values that will change, and will not match up to an HTML page.